src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityCallWarning.kt

```diff
@@ -0,0 +1,155 @@
+package at.hannibal2.skyhanni.features.event.hoppity
+
+import at.hannibal2.skyhanni.events.ConfigLoadEvent
+import at.hannibal2.skyhanni.events.GuiRenderEvent
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.LorenzKeyPressEvent
+import at.hannibal2.skyhanni.events.SecondPassedEvent
+import at.hannibal2.skyhanni.features.inventory.chocolatefactory.ChocolateFactoryAPI
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ColorUtils.toChromaColorInt
+import at.hannibal2.skyhanni.utils.ConditionalUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.HypixelCommands
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.RegexUtils.matches
+import at.hannibal2.skyhanni.utils.SoundUtils
+import net.minecraft.client.Minecraft
+import net.minecraft.client.gui.Gui
+import net.minecraft.client.renderer.GlStateManager
+import net.minecraftforge.fml.common.eventhandler.EventPriority
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import org.lwjgl.input.Keyboard
+import kotlin.math.sin
+import kotlin.time.Duration.Companion.seconds
+
+@SkyHanniModule
+object HoppityCallWarning {
+
+    /**
+     * Test messages (and the real ones from Hypixel) have a space at the end of
+     * them that the IDE kills. So it's "§r§e ✆ "
+     *
+     * REGEX-TEST: §e✆ §r§bHoppity§r§e ✆
+     * REGEX-TEST: §e✆ §r§aHoppity§r§e ✆
+     */
+    private val initHoppityCallPattern by ChocolateFactoryAPI.patternGroup.pattern(
+        "hoppity.call.init",
+        "§e✆ §r(?:§a|§b)Hoppity§r§e ✆.*",
+    )
+
+    /**
+     * REGEX-TEST: §a✆ RING... §r §r§2§l[PICK UP]
+     * REGEX-TEST: §a✆ RING... RING... §r §r§2§l[PICK UP]
+     * REGEX-TEST: §a✆ RING... RING... RING... §r §r§2§l[PICK UP]
+     * REGEX-TEST: §a✆ RING... RING... RING...
+     */
+    private val callRingPattern by ChocolateFactoryAPI.patternGroup.pattern(
+        "hoppity.call.ring",
+        "§a✆ (?:RING\\.{3} ?){1,3}(?:§r §r§2§l\\[PICK UP])?",
+    )
+
+    /**
+     * REGEX-TEST: eaf78cc9-260d-407f-b1df-efea83e5038a
+     * REGEX-TEST: 2bf7445f-2f86-406a-b629-deb5e6e03faa
+     */
+    private val cbUuidPattern by ChocolateFactoryAPI.patternGroup.pattern(
+        "hoppity.call.uuid",
+        "[a-f0-9]{8}-(?:[a-f0-9]{4}-){3}[a-f0-9]{12}",
```

Comment:
Instead use something like this and move it into a utils file
![image](https://github.com/user-attachments/assets/79da5446-e293-4bda-96cb-68e4b72e8924)